notebooks/signac_108_Sacred.ipynb

```diff
@@ -0,0 +1,83 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "specified-jordan",
+   "metadata": {},
+   "source": [
+    "# 1.8 Integration with Sacred\n",
+    "\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "NOTE: These cells are not executable code, but are meant to serve as a  guideline for setting up a Sacred project with signac\n",
+    "    \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "random-review",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Integrating a [sacred]( https://sacred.readthedocs.io/) experiment with **signac-flow** is very simple.\n",
```

Comment:
Re-writing to avoid the phrase "is very simple."
```suggestion
    "Here we demonstrate how to integrate a [sacred](https://sacred.readthedocs.io/) experiment with **signac-flow**.\n",
```